api/jobs/handlers.py

```diff
@@ -0,0 +1,105 @@
+"""
+API request handlers for the jobs module
+"""
+
+from ..dao.containerutil import create_filereference_from_dictionary, create_containerreference_from_dictionary
+
+from .. import base
+from .. import config
+
+from .jobs import Job
+from .queue import Queue
+
+log = config.log
+
+
+class JobsHandler(base.RequestHandler):
+
+    """Provide /jobs API routes."""
+
+    def get(self):
+        """
+        List all jobs.
+        """
+        if not self.superuser_request:
+            self.abort(403, 'Request requires superuser')
+
+        return list(config.db.jobs.find())
```

Comment:
What typically consumes this endpoint?

I tested on my dev environment with a large dataset (8900 jobs) and the request was ~2000 ms, for reference. Depending on what consumes this and how large we expect that container to grow, this might be a good candidate for pagination in the future.